src/contracts/dapp/DAppControl.sol

```diff
@@ -34,6 +34,10 @@ abstract contract DAppControl is DAppControlTemplate, ExecutionBase {
             // Max one of user or dapp nonces can be sequential, not both
             revert AtlasErrors.BothUserAndDAppNoncesCannotBeSequential();
         }
+        if (_callConfig.invertBidValue && _callConfig.exPostBids) {
+            // If both invertBidValue and exPostBids are true, solver's retreived bid cannot be determined
+            revert AtlasErrors.InvertBidValueCannotBeExPostBids();
+        }
```

Comment:
Nice. I think we should add this check to AtlasVerification `validateCalls()` as well because we can't guarantee that dapps use our DAppControl.sol template